extensions/libraries/redcore/language/en-GB/en-GB.lib_redcore.ini

```diff
@@ -116,6 +116,7 @@ LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_REQUEST_HEADER_OPTIONS_DESC="While
 LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_REQUEST_HEADER_OPTIONS_ACCEPT_LANGUAGE="(Optional) Set the language for your request result (ex. en). Default language is %s. Allowed languages are: %s"
 LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_REQUEST_HEADER_OPTIONS_TRANSLATION_FALLBACK="(Optional) Set Translation fallback option to override default API behavior. Default Translation fallback is %s. Allowed options are: true, false"
 LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_REQUEST_HEADER_OPTIONS_OUTPUT_FORMAT="(Optional) Output format option can be changed through this server setting. Default Output format is %s. Allowed formats are: %s"
+LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_REQUEST_HEADER_OPTIONS_STATEFUL = "(Optional)Toggle to determin if the application should remember state between each request. Default is %s. Allowed options is 1, 0"
```

Comment:
spellcheck? :)
(Optional) Toggle to determine if the application should remember state between each request. Default is %s. Allowed options are 1, 0